frame/dvm/ethereum/Cargo.toml

```diff
@@ -28,8 +28,6 @@ libsecp256k1 = { version = "0.3", default-features = false }
 dvm-consensus-primitives = { path = "../consensus/primitives", default-features = false }
 dvm-rpc-primitives = { path = "../rpc/primitives", default-features = false }
 darwinia-support                  = { default-features = false, path = "../../../frame/support" }
-
-[dev-dependencies]
```

Comment:
[dev-dependencies] are only used for tests, can be std, but frame dependencies must be no_std, be careful with ```default-features = false``` when moving between them

Reply:
What you said is too right. It took us a long time to debug the problem it causes.